core/src/main/java/jenkins/management/ConsoleLink.java

```diff
@@ -57,6 +57,6 @@ public String getUrlName() {
 
     @Override
     public Permission getRequiredPermission() {
-        return Jenkins.RUN_SCRIPTS;
+        return Jenkins.ADMINISTER;
```

Comment:
Meaning the override could be deleted I think.

Reply:
```suggestion
```